### PR TITLE
Library.java changes

### PR DESCRIPTION
**Pull request by 9Mcgusto6**
All of the changes originating in Library.java have been made. All variables have been changed to meaningful and proper names. Curly brackets have been added wherever applicable. Consistency in spacing has been applied. Refactor -> Rename has been used to apply changes made in Library.java to other files. 
Comments added to the original line of any change containing what has been replaced and what it was replaced with.

### CODE REVIEW

File: src/library/entities/Library.java [hunk 1]
```diff
@@ -84,163 +84,163 @@ public static synchronized void SaVe() {
...
-		CuRrEnT_LoAnS.remove(bOoK.gEtId());
+		currentLoan.DiScHaRgE();
+		currentLoans.remove(book.gEtId());
 	}
 
-
-	public void cHeCk_CuRrEnT_LoAnS() {
-		for (Loan lOaN : CuRrEnT_LoAnS.values()) 
-			lOaN.cHeCk_OvEr_DuE();
-
```
Comment (Firedingo) [inline src/library/entities/Library.java:230]: For loop needs curly brackets

File: src/library/entities/Library.java [hunk 2]
```diff
@@ -84,163 +84,163 @@ public static synchronized void SaVe() {
...
 
-
-	public void cHeCk_CuRrEnT_LoAnS() {
-		for (Loan lOaN : CuRrEnT_LoAnS.values()) 
-			lOaN.cHeCk_OvEr_DuE();
-				
+	
+	public void checkCurrentLoans() { //Changed cHeCk_CuRrEnT_LoAnS to checkCurrentLoans
+		for (Loan loan : currentLoans.values()) //Changed lOaN to loan
+			loan.cHeCk_OvEr_DuE();
```
Comment (Firedingo) [inline src/library/entities/Library.java]: For Loop Needs Curly Brackets

File: src/library/entities/Library.java [hunk 3]
```diff
@@ -17,65 +17,65 @@
...
+					self = (Library) libraryFile.readObject();
+					Calendar.getInstance().setDate(self.loanDate); //Calendar.gEtInStAnCe().SeT_DaTe changed to getInstance().setDate to match method in Calendar
+					libraryFile.close();
 				}
 				catch (Exception e) {
 					throw new RuntimeException(e);
 				}
 			}
-			else SeLf = new Library();
+			else self = new Library();
```
Comment (Firedingo) [inline src/library/entities/Library.java]: Technically curly brackets can be used here so probably should be.

**Review by Firedingo — CHANGES_REQUESTED** [review]
A Couple of Curly brackets are missing in places they could go in. Otherwise it looks good to me. I've flagged the spots where the curly brackets should be.

File: src/library/entities/Library.java (hunk 2 above)
Comment (9Mcgusto6) [reply src/library/entities/Library.java]: Added curly brackets in the latest commit

File: src/library/entities/Library.java (hunk 3 above)
Comment (9Mcgusto6) [reply src/library/entities/Library.java]: Added curly brackets in latest commit.

**Review by Firedingo — APPROVED** [review]
Looks All Good To Me

**Pull request merged**